### PR TITLE
Feature: Art and Sound Pass for Vandal Imp Tossed Attack [MTT-2732]

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description
An art and sound pass for the Vandal Imp tossed attack! Created a bunch of fancy new VFX and worked with Fernando to implement them for the tossed attack. Also cut up our existing tossed attack sound effect to work with this version. 

**(cherry picked from commit baef02ba6280f7d48eba6e8305b5af24f7340361):** These changes were originally in a branch based off of the [Network Rigid Body branch](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/671) that has since been merged into develop, so to keep things clean, I've cherry picked them into this branch which is based off of develop. 

Here's how it looks now :)

https://user-images.githubusercontent.com/89089503/176210898-df125eed-4174-4afc-83f4-256887b0f0e0.mp4

### Issue Number(s)
[Jira ticket [MTT-2732]](https://jira.unity3d.com/browse/MTT-2732)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Comment by SamuelBellomo** [comment]
Contribution check list says there are tests added?

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 * Updated boss room's root scene to automatically load child scenes at editor time (#653)
 * Users can change profile in-game in addition to the -AuthProfile command line argument (#636)
 * NetworkRigidbody-based toss Action, thrown by new VandalImp class [MTT-2333](#671)
+* Art and sound pass for NetworkRigidbody-based toss action [MTT-2732](#689)
```
Comment (SamuelBellomo) [inline CHANGELOG.md:23]:
```suggestion
* New Vandal Imp and bomb throwing action: NetworkRigidbody-based toss Action, thrown by new VandalImp class [MTT-2333](#671)
  * Art and sound pass for NetworkRigidbody-based toss action [MTT-2732](#689)
```

Comment (SamuelBellomo) [reply CHANGELOG.md:23]: Also this shouldn't be in 1.3.0, should be in unreleased.

**Comment by jilfranco-unity** [comment]
> Contribution check list says there are tests added?

My b, didn't mean to check this! :P

File: Assets/Scripts/Gameplay/GameplayObjects/ServerTossedItem.cs [hunk 2]
```diff
@@ -32,6 +32,9 @@ public class ServerTossedItem : NetworkBehaviour
 
         float m_DetonateAfterSeconds;
 
+        [SerializeField]
+        float destroyAfterSeconds = 6f;
```
Comment (LPLafontaineB) [inline Assets/Scripts/Gameplay/GameplayObjects/ServerTossedItem.cs:36]: naming nitpick, should be m_PascalCase. Same for detonateAfterSeconds above

File: Assets/Scripts/Gameplay/GameplayObjects/ServerTossedItem.cs [hunk 3]
```diff
@@ -98,12 +102,14 @@ void FixedUpdate()
             if (!m_Detonated && m_DetonateAfterSeconds < Time.fixedTime)
             {
                 Detonate();
+            }
 
+            if (m_Detonated && m_DestroyAfterSeconds < Time.deltaTime)
```
Comment (LPLafontaineB) [inline Assets/Scripts/Gameplay/GameplayObjects/ServerTossedItem.cs:107]: why do we use Time.deltaTime here and Time.fixedTime above?

**Review by LPLafontaineB — COMMENTED** [review]
Found a bug while testing. Late-joining clients see all previous bombs on the ground as they were before exploding
image

**Review by LPLafontaineB — DISMISSED** [review]

**Review by fernando-cortez — DISMISSED** [review]
Only nitpick is to place the little rascal somewhere in between the starting area and the boss? Maybe right before the door? He just screams mini-boss to me 😃

**Comment by jilfranco-unity** [comment]
Moved the vandal imp to be closer to the boss door! I also rearranged the surrounding imps a bit so that he fit in better :)

**Review by LPLafontaineB — APPROVED** [review]

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**